addons/mail/static/src/discuss/core/public_web/discuss_sidebar_categories.scss

```diff
@@ -78,17 +73,17 @@ $o-mail-DiscussSidebarChannel-borderOpacity: .05;
     left: 3px;
 }
 
-.o-mail-DiscussSidebarChannel-subChannel.o-nonCompact {
+.o-mail-DiscussSidebarChannel-subChannel {
     margin-left: map-get($spacers, 4) + map-get($spacers, 2);
 }
 
 .o-mail-DiscussSidebar-unreadIndicator {
     font-size: 0.35rem;
-    left: 0px;
+    left: 1px;
```

Comment:
`left: 1px` is wrong: it was changed to `0px` recently and this is the right value. Should not change to prior value 1px which looks worse. Again this is unrelated to the change at hand.